data/events.json

```diff
@@ -0,0 +1,492 @@
+  },
+  {
+    "eventDbId": "6000000053",
+    "studyDbId": "6000000010",
+    "date": {
+      "discreteDates": ["2017-05-12"]
+    },
+    "eventDescription": "Coragen (active ingredient Chlortraniliprole) injected into subsurface drip system in the afternoon, approximately 2:00 PM.  Applied at rate 4.5 oz/ac, for total of approximately 6 oz.",
+    "eventParameters": [
+        {
+          "code": "CH_NAME",
+          "name": "chem_applic_name",
+          "unit": "text",
+          "value": "Coragen"
+        },
+        {
+          "code": "CHCD",
+          "name": "chem_applic_material",
+          "unit": "text",
+          "value": "Chlortaniliprole"
+        },
+        {
+          "code": "CH",
+          "name": "chem_applic_level",
+          "unit": "kg/ha",
+          "value": "2.9"
+        },
+        {
+          "code": "CH_TARGETS",
+          "name": "chemical_applic_target",
+          "value": "grub"
+        }
```

Comment:
needs `unit` 
ICASA says its a code, but I couldnt find the list of encoded values. so maybe just `"unit": "text"`

Reply:
there is a list in the Pest_codes worksheet. It didn't include 'grub' but does include 'Cutworm' which is one of the targets ... 

@BrapiCoordinatorSelby are parameterDescription and codeValueDescription required? Did I get the use correct?

Reply:
I think `parameterDescription` is optional, it is just extra human readable text to explain the parameter. Depending on the use case, parameter, and data available, it may or may not be included.

`codeValueDescription` is also optional but with some restrictions. The intent of `codeValueDescription` is to add extra human readable information to a value that is an ICASA code. 
If `unit` does not equal `"code"`, then `codeValueDescription` should not be included. I believe in your update, there were a few places that broke this rule.
If `unit` does equal `"code"`, then I would strongly suggest including `codeValueDescription`, though not required.

Reply:
thanks!

Reply:
Linking to the [spec](https://brapinewconceptpreview.docs.apiary.io/#/reference/agronomy-and-crop-modeling/get-events) to remind myself where these rules are located.